chef_master/source/cookstyle.rst

```diff
@@ -9,18 +9,160 @@ Most of the code that is authored when working with Chef is written as Ruby. Jus
 
 .. end_tag
 
-cookstyle is a linting tool based on RuboCop designed to address the following issues:
+Cookstyle is a linting tool based on the RuboCop Ruby linting tool. Use Cookstyle to author better cookbook code:
 
-* New releases of RuboCop typically contain new style rules. These often cause projects to fail style checks after the upgrade.
-* The default configuration for RuboCop enables many checks that are inappropriate for cookbook development.
+* Enforce style conventions and best practices
+* Evaluate the code in a cookbook against metrics like "line length" and "function size"
+* Help every member of a team to author similary structured code
+* Estabilish uniformity of source code
```

Comment:
Straight from the Rubocop doc. I'll nuke that entire line since it's not actually necessary